spec/models/account_spec.rb

```diff
@@ -326,8 +326,12 @@
   end
 
   describe '.following_map' do
+    let(:account) { Fabricate(:account) }
+    let(:target) { Fabricate(:account) }
+    let!(:follow) { Fabricate(:follow, account: account, target_account: target) }
+
     it 'returns an hash' do
-      expect(Account.following_map([], 1)).to be_a Hash
+      expect(Account.following_map([target.id], account.id)).to eq({ "#{target.id}".to_i => true })
```

Comment:
Can you explain the `"#{target.id}".to_i` thing ... shouldn't `target.id` already be an integer?

Also - it looks like this change is adding spec coverage (thanks!) and is not just a style fix?  Maybe put it in a separate commit or separate PR?